ephios/core/views/event.py

```diff
@@ -253,17 +253,125 @@ def get_context_data(self, **kwargs):
         ctx["eventtypes"] = EventType.objects.all()
         ctx["filter_form"] = self.filter_form
 
-        if (new_mode := self.request.GET.get("mode")) in {"list", "calendar"}:
+        if (new_mode := self.request.GET.get("mode")) in {"list", "calendar", "day"}:
             self.request.session["event_list_view_mode"] = new_mode
         mode = self.request.session.get("event_list_view_mode", "list")
         ctx["mode"] = mode
 
         if mode == "calendar":
             ctx.update(self._get_calendar_context())
+        elif mode == "day":
```

Comment:
Please extract the day branch to its own method like with calendar.

Reply:
Done